src/app/parcels/batch/BatchParcelDataGrid.tsx

```diff
@@ -200,7 +202,15 @@ export const defaultTableState: BatchTableDataState = {
 };
 
 const BatchParcelDataGrid: React.FC<BatchParcelDataGridProps> = ({ initialTableState }) => {
-    const [tableState, _] = useReducer(batchParcelsReducer, initialTableState);
+    const dataGridInitialLoad = useRef(true);
+    const [tableState, dispatch] = useReducer(batchParcelsReducer, initialTableState);
+
+    useEffect(() => {
+        if (dataGridInitialLoad.current) {
+            dispatch({ type: "initialise_table_state" });
+            dataGridInitialLoad.current = false;
+        }
+    }, []);
```

Comment:
If this is the only use of the dataGridInitialLoad ref then maybe see if you can extract the useEffect elsewhere? The use of the ref feels a little strange